openc3/bin/openc3cli

```diff
@@ -648,6 +650,59 @@ def run_bridge(filename, params)
   end
 end
 
+def await_dotter(script_id)
+  OpenC3::RunningScriptWebSocketApi.new(id: script_id) do |api|
+    while (resp = api.read) do
+      case resp['type']
+      when 'file'
+        puts resp.pretty_inspect
+      when 'line'
+        fn = resp['filename'].nil? ? '<no file>' : resp['filename']
+        puts "At [#{fn}:#{resp['line_no']}] state [#{resp['state']}]"
+      when 'output'
+        puts resp['line']
+      when 'time'
+        puts resp.pretty_inspect
+      when 'complete'
+        puts 'script complete'
+        break
+      else
+        puts resp
+      end
+      sleep(2)
+    end
+  end
+end
+
+def await_script(args=[])
+  #  we are limiting the wait for status, not the script run time
+  #  we want 0 to mean 'forever'
+  #  REVIEW:       do we want to accept suite, scope, and environment options at the cli?
+  wait_limit = 0
+  if args[0] =~ /^--d*/
+    wait_limit = args[0][2..-1].to_i
+    args.shift
+  end
+  abort("No script file provided") if args[0].nil?
+  require 'openc3/script'
+  # REVIEW: is there a case to be made for requiring script/ once at the top of this file?
```

Comment:
Yeah just put it in the top require list

Reply:
Done openc3cli still runs for me, but I didn't test the other affected code paths.